commerce_coordinator/apps/commercetools/tasks.py

```diff
@@ -254,3 +270,67 @@ def refund_from_mobile_task(
             f"with error {err.errors} and correlation id {err.correlation_id}"
         )
         raise err
+
+
+@shared_task(
+    autoretry_for=(CommercetoolsError,),
+    retry_kwargs={"max_retries": 5, "countdown": 3},
+)
+def revoke_line_mobile_order_task(payment_id: str):
```

Comment:
Added monitor for the failure of this task: https://app.datadoghq.com/monitors/174620081